src/rgw/rgw_sal_motr.cc

```diff
@@ -4155,14 +4159,14 @@ int MotrMultipartUpload::complete(const DoutPrefixProvider *dpp,
         compressed = true;
       }
 
-      // We may not need to do the following as remove_objs are those
+#if 0 // We may not need to do the following as remove_objs are those
```

Comment:
if this is dead code, why can't we remove it?

Reply:
We can remove it. I don't see issue with that.

Reply:
Done